<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceMenuEditorForm</class>
 <widget class="QWidget" name="ChoiceMenuEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditMenuItem">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxJump">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>JumpTo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSetVar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Variable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu Item:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditJumpTo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVarSet">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddOption">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AddMenuItem&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelOption">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;RemoveMenuItem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFinished">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Finished</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetChoices">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>256</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDialog">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDialog">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Dialog</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
